### PR TITLE
 Add Version Id

### PR DESCRIPTION
**Pull request by rpmoore**
This adds the version id to the output of the get_objects_on_tape command.

### CODE REVIEW

**Comment by shabtaisharon** [comment]
you didn't need to update the test to reflect this change?

**Comment by rpmoore** [comment]
I probably should update the test...

File: ds3_java_cli/src/test/java/com/spectralogic/ds3cli/Ds3Cli_Test.java [hunk 1]
```diff
@@ -2087,62 +2084,16 @@ public void getBlobsOnTape() throws Exception {
...
-                "| coffeehouse/witchcraft.mp3                      | coffeehouse |   6409093 | 92a40cff-63a6-4520-81a9-80afa03a1973 |\n" +
-                "+-------------------------------------------------+-------------+-----------+--------------------------------------+\n";
+                "+--------------------+-------------+------+--------------------------------------+--------------------------------------+\n" +
+                "|        Name        |    Bucket   | Size |                  Id                  |              Version-Id              |\n" +
+                "+--------------------+-------------+------+--------------------------------------+--------------------------------------+\n" +
+                "| 123456789.txt      | coffeehouse |    9 | 53452a07-699a-4c27-8de5-95aa0a431df1 | 53452a07-699a-4c27-8de5-95aa0a431df1 |\n" +
+                "| Always_295x166.jpg | coffeehouse | 9172 | 7989ad4a-47a5-41ac-8814-3746e4e20679 | 7989ad4a-47a5-41ac-8814-3746e4e20679 |\n" +
+                "+--------------------+-------------+------+--------------------------------------+--------------------------------------+\n";
```
Comment (shabtaisharon) [inline ds3_java_cli/src/test/java/com/spectralogic/ds3cli/Ds3Cli_Test.java]: why does the output is so short now? i don't see you change the number files of the test

File: ds3_java_cli/src/test/java/com/spectralogic/ds3cli/Ds3Cli_Test.java [hunk 2]
```diff
@@ -2087,62 +2084,16 @@ public void getBlobsOnTape() throws Exception {
...
 
         final String packet = "<Data>" +
                 "<Object Bucket=\"coffeehouse\" Id=\"53452a07-699a-4c27-8de5-95aa0a431df1\" Latest=\"true\" Length=\"9\" Name=\"123456789.txt\" Offset=\"0\" VersionId=\"53452a07-699a-4c27-8de5-95aa0a431df1\"/>" +
                 "<Object Bucket=\"coffeehouse\" Id=\"7989ad4a-47a5-41ac-8814-3746e4e20679\" Latest=\"true\" Length=\"9172\" Name=\"Always_295x166.jpg\" Offset=\"0\" VersionId=\"7989ad4a-47a5-41ac-8814-3746e4e20679 \"/>" +
-                "<Object Bucket=\"coffeehouse\" Id=\"6649c2cb-6e83-4c58-9fb8-9b4aec8b014b\" Latest=\"true\" Length=\"29895\" Name=\"Chapter 9.docx\" Offset=\"0\" VersionId=\"6649c2cb-6e83-4c58-9fb8-9b4aec8b014b\"/>" +
```
Comment (shabtaisharon) [inline ds3_java_cli/src/test/java/com/spectralogic/ds3cli/Ds3Cli_Test.java]: same question here

File: ds3_java_cli/src/test/java/com/spectralogic/ds3cli/Ds3Cli_Test.java (hunk 1 above)
Comment (rpmoore) [reply ds3_java_cli/src/test/java/com/spectralogic/ds3cli/Ds3Cli_Test.java]: I removed the extra entries from the test to simplify modifying it.  I changed the xml input, and the table formatted response.

**Pull request merged**